h5c/vic/src/vic-webapp/src/app/create-vch-wizard/create-vch-wizard.component.ts

```diff
@@ -36,6 +36,8 @@ export class CreateVchWizardComponent implements OnInit {
   public errorMsgs: string[];
   private _cachedData: any = {};
   private cloneTicket = '';
```

Comment:
If we are not going to reuse these private variables let's remove them and declare local variables inside the `.subscribe()` call

Reply:
If there's a value in reusing these private variables later let's follow the convention of prefixing these with "_"